### PR TITLE
Stack Sheets Cancel Button

### PR DESCRIPTION
**Pull request by Davidj361**
[Changelogs]:

:cl:
tweak: Stacked sheets now have a cancel button when taking an amount from them.
/:cl:

[why]:
To cancel taking sheets before, you would have to type 0.

### CODE REVIEW

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -339,7 +339,7 @@
 			return
 		//get amount from user
 		var/max = get_amount()
-		var/stackmaterial = round(input(user,"How many sheets do you wish to take out of this stack? (Maximum  [max])") as num)
+		var/stackmaterial = round(input(user,"How many sheets do you wish to take out of this stack? (Maximum  [max])") as null|num)
 		max = get_amount()
 		stackmaterial = min(max, stackmaterial)
 		if(stackmaterial == null || stackmaterial <= 0 || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
```
Comment (vuonojenmustaturska) [inline code/game/objects/items/stacks/stack.dm]: can you replace this null check with isnull() while you're here

**Review by vuonojenmustaturska — APPROVED** [review]

**Comment by PKPenguin321** [comment]
I love this pr thanks

**Comment by vuonojenmustaturska** [comment]
whatever

**Comment by JohnGinnane** [comment]
thank you very much for this

**Pull request merged**